api/src/middleware/__tests__/programsRouter.ts

```diff
@@ -211,4 +211,48 @@ describe('programsRouter', () => {
         .expect(500, done);
     });
   });
+
+  describe('GET /activityStatus/:programId', () => {
```

Comment:
The tests should appear in the same order in the test file that the routes do in the router middleware file; in the middleware file, your route is defined *above* the route that specifies both the `programId` and the `activityId`; you should move your tests (defined starting on line 215 of this file) *above* the tests for the route that specifies both the `programId` and `activityId`.